### PR TITLE
BETA: Register nidun.is-a.dev

### PR DESCRIPTION
**Pull request by nldun**
Added `nidun.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by Omxpro — CHANGES_REQUESTED** [review]
we cannot review your site as it redirects to is-a.dev. please remove the domain so we can review it. and also i'd recooment switching URL record to CNAME. read more about it at https://is-a.dev/docs

**Comment by nldun** [comment]
my misteke to close that

**Comment by nldun** [comment]
how about now a helper?

File: domains/nidun.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "new update to request new cname",
+    "repo": "https://github.com/nldun/ininidun.github.io",
+    "owner": {
+        "username": "nldun",
+        "email": "rezkyto@gmail.com",
+    },
+    "record": {
+        "CNAME": "ininidun.github.io"
```
Comment (CuteDog5695) [inline domains/nidun.json:9]: website doesnt exist (404 page)

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/nidun.json (hunk 1 above)
Comment (CuteDog5695) [reply domains/nidun.json:9]: tho if you do have website hosted post the link here and if its good ill merge it

File: domains/nidun.json [hunk 2]
```diff
@@ -6,6 +6,6 @@
         "email": "rezkyto@gmail.com",
     },
     "record": {
-        "CNAME": "ininidun.github.io"
+        "CNAME": "nldun.github.io/ininidun.github.io"
```
Comment (CuteDog5695) [inline domains/nidun.json:9]: cant have / in CNAME so remove / and the stuff after it

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/nidun.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "new update to request new cname",
+    "repo": "https://github.com/nldun/ininidun.github.io",
+    "owner": {
+        "username": "nldun",
+        "email": "rezkyto@gmail.com",
```
Comment (CuteDog5695) [inline domains/nidun.json:6]: yea you forgot to remove the comma at line 6, please remove that

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/nidun.json (hunk 3 above)
Comment (nldun) [reply domains/nidun.json:6]: okay

Comment (nldun) [reply domains/nidun.json]: done i remove the cOMMAAAA sirr

**Review by iostpa — APPROVED** [review]
seems good to me now

**Pull request merged**